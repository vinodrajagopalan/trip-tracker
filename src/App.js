import React from "react";
import { hot } from "react-hot-loader/root";

class App extends React.Component {
  render() {
    const { name } = this.props;
    return (
      <>
        <h1>Hello world1 {name}</h1>
      </>
    );
  }
}

export default hot(App);
