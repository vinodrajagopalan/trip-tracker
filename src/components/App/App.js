import React from "react";
import Proptypes from "prop-types";
import { hot } from "react-hot-loader/root";
import { Title } from "./App.style";

class App extends React.Component {
  render() {
    const { name } = this.props;
    return (
      <>
        <Title>{name}</Title>
      </>
    );
  }
}

App.propTypes = {
  name: Proptypes.string.isRequired,
};

export default hot(App);
