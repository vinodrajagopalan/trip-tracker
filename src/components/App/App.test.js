import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

describe("App Component", function () {
  it("should have hello world message", function () {
    let { getByText } = render(<App name="hello world"/>);
    expect(getByText("hello world")).toMatchSnapshot();
  });
});
