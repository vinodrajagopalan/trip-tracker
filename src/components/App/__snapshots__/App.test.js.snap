// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component should have hello world message 1`] = `
<h1
  class="sc-bdvvtL gEFSUE"
>
  hello world
</h1>
`;
